EAP-TLS_Solution/WebApiClient/.gitignore

```diff
@@ -0,0 +1,5 @@
+/.vs
+/obj
+/bin
+/*.user
+/*.sln
```

Comment:
You and James discussed an issue about .sln files.  Should this be here?  Also, should have a newline at the end.